web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -53,6 +54,14 @@ function dpl_react_apps_preprocess_page(array &$variables): void {
  * Implements hook_dpl_react_apps_data().
  */
 function dpl_react_apps_dpl_react_apps_data(array &$data): void {
+  $adgangsplatformen_config = \Drupal::service('dpl_login.adgangsplatformen.config');
+
+  $data['configs'] += [
+    'agency' => json_encode([
+      'id' => $adgangsplatformen_config->getAgencyId(),
+    ]),
+  ];
```

Comment:
Consider if there is a reason to make this an object. It seems like we could use a string which is simpler.